quick_start.adoc

```diff
@@ -26,6 +26,42 @@ We will first walk through setting up your managed cluster set and placement, th
 
 TODO Chris Ahl please add details here, including bringing clusters under management of a hub.
 
+[#creating-auth-realm]
+== Creating your AuthRealm
+
+The AuthRealm custom resoure specifies a combination of one or more https://docs.openshift.com/container-platform/4.8/authentication/understanding-identity-provider.html[identity providers] and a placement that determines to which managed clusters the identity provider(s) should be applied.
+
+An AuthRealm resource example is as follows:
+[source,yaml]
+----
+apiVersion: identityconfig.identitatem.io/v1alpha1
+kind: AuthRealm
+metadata:
+  name: identity-config-developers
+  namespace: identity-config-authrealm
+spec:
+  identityProviders:
+  - github:
+      clientID: 41c30d83fd1b4774f871
+      clientSecret:
+        name: identitatem-github-client-secret
+      organizations:
```

Comment:
We might want to not have the organizations section in the example as we did see some issue with it in and less problems with it not specified.

Reply:
@vidyanambiar has been researching and is going to add explanation on how to make it work.

Reply:
ok